layers/+readers/pdf/packages.el

```diff
@@ -135,9 +154,9 @@
       (spacemacs/set-leader-keys-for-major-mode 'pdf-occur-buffer-mode
         "tf" 'next-error-follow-minor-mode))))
 
-(defun pdf/init-pdf-view-restore ()
-  (use-package pdf-view-restore
-    :after pdf-tools
-    :defer t
-    :init
-    (add-hook 'pdf-view-mode-hook 'pdf-view-restore-mode)))
+;; (defun pdf/init-pdf-view-restore ()
```

Comment:
why is this commented out?

Reply:
Ah... that is an accident. I use this locally, while I am working on implementing/improving continuous scroll in pdf-tools...